ads/inabox/inabox-messaging-host.js

```diff
@@ -129,6 +129,15 @@ export class InaboxMessagingHost {
       return false;
     }
 
+    const allowedTypes =
+        this.iframeMap_[request['sentinel']].iframe.dataset['ampAllowed'];
```

Comment:
Let's refactor this so you can rely on the output of getFrameElement by doing the following:

- change getFrameElement_ to return ?AdFrameDef
- you can then do const allowedTypes = adFrame.iframe.dataSet['ampAllowed'];
- modify iframe usage below to be adFrame.measurableFrame

Reply:
Changed.